runtime/parser/schema/project.schema.json

```diff
@@ -203,54 +226,55 @@
       }
     },
     "api": {
-      "title": "api",
       "type": "object",
+      "title": "API YAML",
+      "description": "In your Rill project directory, create a new file name `<api-name>.yaml` in the `apis` directory containing a custom API definition. See comprehensive documentation on how to define and use [custom APIs](/integrate/custom-apis/index.md)",
       "examples": [
         {
+          "_comment_heading": "Example: This api returns the top 10 authors by net line changes since the specified date provided in the arguments.",
           "type": "api",
-          "connector": "clickhouse",
-          "sql": "select * from my_model"
+          "name": "metrics_view_api",
+            "metrics_sql": "SELECT author_name, net_line_changes\nFROM advanced_metrics_view\n  where author_date > '{{ .args.date }}'\n  order by net_line_changes DESC\n  limit 10"
```

Comment:
I still don't understand this way of writing examples in JSON schema. It feels quite weird to break out a YAML snippet in separate entries like this. I would have expected just a single string with line breaks and similar using escape characters like `\n`.

If you are sure this is the recommended way, can you share a link to the place in the JSON schema spec that recommends this?